test/jdk/com/apple/laf/RootPane/RootPaneDefaultButtonTest.java

```diff
@@ -128,7 +128,7 @@ private static void test(Robot robot, AbstractButton buttonToClick,
         robot.delay(20);
         robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
 
-        robot.delay(100);
+        robot.delay(1000);
```

Comment:
it will be useful to add the same delay before the first call to test, right after invokeAndWait.....setVisible(...)
Also please move creation and access_to all Swing components to EDT(button, radioButton1)

Reply:
I added an additional delay. I'm not sure exactly what refactor you're looking for, but I rewrote the test to put more in the EDT.

(I double-checked that the test still passes, but that doesn't prove much since I've never reproduced a failure.)

I did try at one point blocking the testing thread until the EDT had a chance to complete an invokeLater runnable, but that didn't seem to make a difference so I didn't commit it.

Reply:
The code in test() which access swing components(various buttons) also should be moved to EDT.

Reply:
I'm not sure I agree, but that becomes moot after https://github.com/openjdk/jdk/pull/25244/commits/9f91249a424b190990fc3056644b5aac8caa0a5b